libs/wingsdk/src/target-sim/api.ts

```diff
@@ -41,6 +41,17 @@ export class Api extends cloud.Api implements ISimulatorResource {
     this.policy = new Policy(this, "Policy", { principal: this });
   }
 
+  /** @internal */
+  public get _liftMap(): LiftMap {
+    const lifts: Array<[IResource, Array<string>]> = [];
+    for (const handler of Object.values(this.handlers)) {
+      lifts.push([handler.func, [cloud.FunctionInflightMethods.INVOKE]]);
+    }
+    return {
+      [cloud.ApiInflightMethods.REQUEST]: lifts,
+    };
+  }
```

Comment:
What happens if this isn't here? `addEndpoint` Seems to take care of adding a connection between this resource and the underlying function

Reply:
Mmmm - yeah I guess there's some double duty being performed. I tried this and the connection still appears correctly when I test this out in Wing Console locally 👍  so removing for now